info/info.json

```diff
@@ -0,0 +1,954 @@
+      }
+    },
+    "/beta/bike_lanes" : {
+      "get" : {
+        "tags" : [ "Cycling" ],
+        "summary" : "Get bike lanes",
+        "description" : "Get bike lanes, including temporary ones, as a GeoJSON",
+        "operationId" : "getBikeLanes",
+        "parameters" : [ {
+          "name" : "x-access_key",
+          "in" : "header",
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "Information available related to the given input",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/BikeLaneGeoJSON"
+                }
+              }
+            }
+          },
+          "403" : {
+            "description" : "This method requires an `x-access_key` and it is either missing or invalid, or the the credentials provided are not sufficient for access to the specified resource.",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "error"
+                }
+              }
+            }
+          },
+          "500" : {
+            "description" : "Unexpected error",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/coverage" : {
+      "get" : {
+        "tags" : [ "Shared" ],
+        "summary" : "All supported areas",
+        "description" : "All the coverage areas that this operator serves, including optionally (but highly recommended!) information on products and prices. Information about each area is returned as a GeoJSON Feature, within a single GeoJSON FeatureCollection.",
+        "operationId" : "coverage",
+        "parameters" : [ {
+          "name" : "x-access_key",
+          "in" : "header",
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "Detailed information on the coverage of the provider.",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "$ref" : "#/components/schemas/Coverage"
+                }
+              }
+            }
+          },
+          "403" : {
+            "description" : "This method requires an `x-access_key` and it is either missing or invalid, or the the credentials provided are not sufficient for access to the specified resource.",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "error"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/provider" : {
+      "get" : {
+        "tags" : [ "Shared" ],
+        "summary" : "Information of the provider",
```

Comment:
What is a provider in the context of an info connector? Would be good to explain. Is it the data provider as there's no TSP?